ansible/roles-studentvm/studentvm-bash-prompt/tasks/main.yml

```diff
@@ -0,0 +1,40 @@
+---
+- name: Set up studentvm_bash_prompt combined dictionary
+  set_fact:
+    studentvm_bash_prompt: >-
+      {{ studentvm_bash_prompt_defaults
+       | combine(studentvm_bash_prompt_vars    | default( {} ),
+                 studentvm_bash_prompt_secrets | default( {}), recursive=true )
+      }}
+- name: Print combined role variables
+  debug:
+    var: studentvm_bash_prompt
+    verbosity: 2
+
+- name: Install bash-git-prompt
+  become: true
+  block:
+  - name: Download bash-git-prompt
+    git:
+      repo: https://github.com/magicmonty/bash-git-prompt.git
+      dest: "/home/{{ studentvm_bash_prompt.user_name }}/.bash-git-prompt"
```

Comment:
This is a little confusing. The default that is set here is an email address which might error out if it is not a user on the system. Can this reference something like `{{ student_name }}`? Or better, in the defaults for the role.

Reply:
Sample vars:
Set studentvm_bash_promt_vars.user_name: {{ student_name }}

Reply:
ah, i missed that. this is what you get with so many files.